src/components/Tables/Table.cy.tsx

```diff
@@ -67,14 +72,14 @@ const data: Datum[] = [
 
 const smallerData = data.slice(0, 3);
 
-const headers: [string, string][] = [
+const headers = [
     ["full_name", "Name"],
     ["phone_number", "Phone Number"],
-];
+] as const;
 
 interface TestTableProps {
     checkboxes?: boolean;
-    toggleableHeaders?: string[];
+    toggleableHeaders?: (keyof TestData)[];
 }
 
 const Component: React.FC<TestTableProps> = ({ checkboxes = true, toggleableHeaders }) => {
```

Comment:
Why do you have to passs in checkboxes and toggleableHeaders as a prop? I don't see you feeding in checkboxes and toggleableHeaders in any of the `<Component />` that you have mounted below?

Reply:
Ah right I hadn't pushed those changes yet

Reply:
They are now used